### DISCUSSION TITLE
Add "Coins Reserved" tab for Wallet with coins lock option

### DISCUSSION THREAD

**Discussion by brootle** [deso-protocol/dips#78] [Medium CIPs]
Add "Coins Reserved" tab for Wallet, so users can move some of their assets into that category and use as their long term storage. "Reserve" can be added to drop down menu as seen in screenshot. Some interesting functionality can be added for Reserved Coins, like let's say they can be locked for 60 days and user can't buy/sell/transfer during that period. Anyway, we really need separate locating to keep coins that we care to hold for long time. What do you guys think about it?

![image](https://user-images.githubusercontent.com/20090030/125912830-55495957-bbdc-481d-a219-649441fb99a7.png)

**Comment by tijno** [comment]
Why not just use another account ? Use a 2nd account for long term storage is much safer as well.

**Reply by brootle** [reply]
> Why not just use another account ? Use a 2nd account for long term storage is much safer as well.

I want to keep everything in one place

**Reply by tijno** [reply]
Fair point

**Comment by HPaulson** [comment]
Using a cold storage makes much much more sense here, especially security wise. I do think, however, we need an easy way on-site to generate a keypair from a seed phrase without that seed staying in the browser

**Reply by tijno** [reply]
Yes agree - cold storage is the ultimate secure storage for sure. 

Note kronos wallet has a way to generate seed offline.

**Comment by brootle** [comment]
guys really, I just want to add new feature, so I don't need to use hacks like managing 500 accounts, I want to keep everything in one place, simple like that, you want to spread your assets across multiple accounts? ok go ahead, but let me keep my assets nicely organized in one wallet, thank you!

**Comment by ShhubhamA** [comment]
I love this ! This is cool! I like keeping things in one place as its easier and faster to manage! A feature like this would really beneift

**Comment by tijno** [comment]
I think we will see features that may make this easier in future:

Staking - eg Lock some of your clout away to stake.
When you bid on an NFT it will lock the $CLOUT of your bid.

You can check the NFT code about how it "reserves" your clout so you cant spend it until you loose the bid.

**Reply by ShhubhamA** [reply]
Yeah staking is cool but It would take time to withdraw i guess .The feature idea by @brootle is to sort things cause most of us want to look at our creator coin investments with two perspective (Short term gains or long term hodl) Its like even if the reserve investments goes very down and we are wrong in our call ! we can immediately sell and lose less and its not possible with staking !

What Brootle suggests is trader friendly imo

**Reply by tijno** [reply]
Valid point - I had not considered this as part of coin locking.

Re staking - depends on the rules applied to staking - some PoS coins allow you to remove staking at any time. But thats for a separate discussion I guess.

Coin locking is essential feature that kinda was promised but I had a look at the kanban board and I cant spot it there.

Maybe @brootle it would be better to rename this proposal `Lock Creator Coins` as I think it better describes what this is about?

**Reply by brootle** [reply]
renamed into -> Add "Coins Reserved" tab for Wallet with coins lock option, reserve and lock should be synonyms in English?
